tests/build_tests/super_errors/expected/object_literal_passed_when_record_expected.res.expected

```diff
@@ -9,7 +9,9 @@
 
   This has type: [1;31m{"one": bool}[0m
   But it's expected to have type: [1;33mx[0m
-
-  You're passing a [1;31mReScript object[0m where a [1;33mrecord[0m is expected. 
-
-  - Did you mean to pass a record instead of an object? Objects are written with quoted keys, and records with unquoted keys. Remove the quotes from the object keys to pass it as a record instead of object.
\ No newline at end of file
+  
+  You're passing a [1;31mReScript object[0m where a [1;33mrecord[0m is expected. Objects are written with quoted keys, and records with unquoted keys.
+  
+  Possible solutions: 
+  - Rewrite the object to a record, like: [1;33m{one: true}
+[0m
```

Comment:
Another example of how we can suggest an actual and correct rewrite here, since we now have the relevant expression to look at.

Reply:
this is great too! We could also do the same thing for mismatch of objects vs dicts now that we have dict literals, they could become quite common for configs.